create_blog_post_using_azure_openai_and_publish_in_medium/main.bal

```diff
@@ -0,0 +1,44 @@
+import ballerina/io;
+import ballerinax/azure.openai.text;
+import ballerinax/medium;
+
+configurable string openAIToken = ?;
+configurable string serviceUrl = ?;
+configurable string deploymentId = ?;
+configurable string mediumToken = ?;
+
+const API_VERSION = "2023-03-15-preview";
+
+public function main(string title) returns error? {
+    final medium:Client medium = check new ({auth: {token: mediumToken}});
+    final text:Client azureOpenAI = check new (
```

Comment:
Move the initialization just before the first invocation line. i.e. Line 30

Reply:
Addressed in [e6e58df](https://github.com/ballerina-guides/ai-samples/pull/62/commits/e6e58df92842390342212840cced261e85e5fd8e)